ipatests/test_integration/test_installation.py

```diff
@@ -2123,3 +2123,69 @@ def test_hostname_matching_domain(self):
         assert result.returncode == 1
         assert 'hostname cannot be the same as the domain name' \
             in result.stderr_text
+
+
+class TestNsslapdIgnoreTimeSkew(IntegrationTest):
+    """
+    Test to check nsslapd-ignore-time-skew is not disabled.
+    """
+    num_replicas = 1
+    topology = 'line'
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(
+            cls.master, setup_dns=True, extra_args=['--no-dnssec-validation']
+        )
+        tasks.install_replica(
+            cls.master,
+            cls.replicas[0],
+            setup_dns=True,
+            setup_ca=True,
+            extra_args=['--no-dnssec-validation']
+        )
+
+    @pytest.fixture
+    def update_time_skew(self):
+        """
+        Fixture enables nsslapd-ignore-time-skew
+        parameter and reverts it back
+        """
+        ldap = self.replicas[0].ldap_connect()
+        dn = DN(
+            ("cn", "config"),
+        )
+        entry = ldap.get_entry(dn)
+        entry.single_value["nsslapd-ignore-time-skew"] = 'on'
+        ldap.update_entry(entry)
+
+        yield
+
+        entry = ldap.get_entry(dn)
+        entry.single_value["nsslapd-ignore-time-skew"] = 'off'
+        ldap.update_entry(entry)
+
+    def test_check_nsslapd_ignore_time_skew(self, update_time_skew):
+        """
+        This testcase checks that ignore time
+        skew message is displayed in the replica
+        install log and the test also checks that the
+        value of 'nsslapd-ignore-time-skew" is not
+        set to "off" after running the "ipa-replica-manage
+        force-sync" command on the replica
```

Comment:
I would split this test into 2 different test methods:
- test_replica_ca_ignore_time_skew: check that the replica CA installation is done with ignore-time-skew = on (check that the message is present in the replica install logs)
- test_forcesync_does_not_overwrite_ignore_time_skew: check that calling ipa-replica-manage force-sync does not overwrite the value of ignore time skew